.travis.yml

```diff
@@ -16,13 +17,13 @@ script:
 matrix:
     fast_finish: true
 
-install:
-    - composer update $COMPOSER_FLAGS
-
 before_install:
     # disable xdebug if not coverage
     - phpenv config-rm xdebug.ini || echo "xdebug not available"
 
+before_script:
+    - travis_retry composer update --no-progress --prefer-dist $COMPOSER_FLAGS
```

Comment:
There is section `install` for installing all dependencies. Why do you want to change it?

Reply:
I was a bit lost as to why windows didn't launch composer update so I copied it somewhere else. Now I think I understand a little better. "install" was overwrited in windows section so the global install didn't work. The fact that it is now in "before_script" make it launch everywhere.

If you don't like it, I think I can revert that and manually add a line in "install" on the windows block only to ensure composer runs

Reply:
That was one of my assumption.

Just add it to windows > install section